app/controllers/redirection_controller.rb

```diff
@@ -16,6 +16,22 @@ def redirect_latest
     redirect_to(finder_path("search/all", params: { order: "updated-newest" }.merge(redirect_params)))
   end
 
+  def redirect_consultations
+    topics = {}
+    topics["level_one_taxon"] = params[:topics] if params[:topics]
+    topics["level_two_taxon"] = params[:subtaxons] if params[:subtaxons]
+
+    redirect_params = params.merge(topics)
+                            .slice(:departments, :level_one_taxon, :level_two_taxon, :world_locations, :content_store_document_type)
+                            .permit(:level_one_taxon, :level_two_taxon, departments: [],  world_locations: [], content_store_document_type: [])
+                            .transform_keys { |k| k == "departments" ? "organisations" : k }
+                            # .transform_values { |v| v == (["open_consultations"] || ["closed_consultations"] || ["policy_papers"]) ? %w[open_consultations closed_consultations] : v }
```

Comment:
You probably want to delete this commented line from the first commit.

Reply:
👍